nixos/modules/services/x11/desktop-managers/emulationstation.nix

```
@@ -0,0 +1,48 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.xserver.desktopManager.emulationstation;
```

What has this to do with xserver?

```suggestion
  cfg = config.services.desktopManager.emulationstation;
```

---

Aren't all desktopManagers under services.xserver section?